crates/ruff_minimizer/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "ruff_minimizer"
```

Comment:
This is called `shrinking` in property based testing https://www.educative.io/answers/what-is-shrinking

Reply:
that link is helpful, thanks!